rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -768,7 +768,10 @@ namespace rsx
 
 		method_registers[NV4097_SET_LINE_WIDTH] = 1 << 3;
 
-		method_registers[NV4097_SET_FOG_MODE] = 0x0800; // rsx::fog_mode::exponential;
+		// These defaults were found using After Burner Climax (which never set fog mode despite using fog input)
+		method_registers[NV4097_SET_FOG_MODE] = 0x2601; // rsx::fog_mode::linear;
+		(f32&)method_registers[NV4097_SET_FOG_MODE] = 1.;
```

Comment:
Does this fix the [glitchy fog](http://puu.sh/nuEgZ/1633e7032c.png) at the end of the every level/stage that would be seen?